test/NumberConverterTest.java

```diff
@@ -0,0 +1,63 @@
+public class NumberConverterTest {
+
+    public static void main(String[] args) {
+        NumberConverterTest.inttostrTest();
+        NumberConverterTest.strtointTest();
+        NumberConverterTest.dbltostrTest();
+        NumberConverterTest.strtodblTest();
+
+    }
+
+    public static void comparison(String result1, String result2,String iftrue, String ifrail) {
+        if (result2.equals(result1)){
+            System.out.println(iftrue);
```

Comment:
here return to previous variant System.out.println("method strtoint test is successful");
but, the method's name insert using String.format()